input/otlp/logs.go

```diff
@@ -135,6 +135,8 @@ func (c *Consumer) convertLogRecord(
 			eventDomain = v.Str()
 		case "session.id":
 			event.Session.ID = v.Str()
+		case "lifecycle.state":
+			setLabel(k, &event, ifaceAttributeValue(v))
```

Comment:
Why do we need special handling for `lifecycle.state`?

Reply:
Maybe I'm a little confused about how new attributes get added to ECS, but this was to prevent the `label` prefix from being prepended.

Reply:
Nope, you're calling `setLabel` so it will end up as `labels.lifecycle.state`.

At the moment there's several things that are needed to add a top-level field, and map data to it:
 1. Follow https://github.com/elastic/ecs/blob/main/CONTRIBUTING.md to propose the addition to ECS. I guess OTel semantic conventions now?
 2. Add the field to relevant data streams in https://github.com/elastic/apm-server/tree/main/apmpackage
 3. Update https://github.com/elastic/apm-data/tree/main/model and https://github.com/elastic/apm-data/tree/main/model/proto to add the new field to our model types.
 4. Update this code to map the OTel attribute to the field in our model.

Perhaps move this to a new PR once step 1 has happened?